DashAI/front/src/api/run.ts

```diff
@@ -0,0 +1,23 @@
+import api from "./api";
+import type { IRun } from "../types/run";
+
+export const getRuns = async (experimentId: string = ""): Promise<IRun[]> => {
+  const params = experimentId !== "" ? { experiment_id: experimentId } : {};
```

Comment:
Is it possible that experimentID is an empty string?

In other words, is it possible for any run not to have an associated experiment?